nicegui/observables.py

```diff
@@ -111,79 +111,88 @@ def __iadd__(self, other: Any) -> Any:
 
 class ObservableSet(set):
 
-    def __init__(self, data: set, on_change: Callable) -> None:
+    def __init__(self, data: set = set(), on_change: Callable = None) -> None:
```

Comment:
The default value `set()` is very dangerous, because all observable sets that use the default value will share the same instance. Why do we need this default value anyway?

Besides that: The type annotation for `on_change` should be `Optional`.

Reply:
I thought `set()` creates a new instance every time. I introduced it to make the usage as simple as possible (eg. `self.urls = ObservableSet()` instead of `self.urls = ObservableSet(set=set())`).

Reply:
Ah now I got it. Static initalizers are bad. As @falkoschindler suggested I now initalize with None and set the value inside the constructor.